### ISSUE TITLE
Feature: one-time scan on click on GUI

### ISSUE THREAD

**Issue by jakubjosef** [AHAAAAAAA/PokemonGo-Map#2314]
## Expected Behavior
It will be nice if users can trigger a one-time scan in GUI, much better than continuous scanning implemented now. I dont need to run scanner, if i don't using map application right now.

## Current Behavior
Scanning is continuous and cannot be disabled or enabled.

## Possible Solution
Add possibility (by runtime param) to disable continuous scanning and enable user triggered onetime scanning.

**Comment by Zerotask** [comment]
The same like Pokevision. That's would be cool.

**Comment by man0s** [comment]
That would be awesome, as for network.. less CPU too!

**Comment by Exoyds** [comment]
Agreed! I actually filed a feature request for that a while back! But imo you should be able to run both continuous scanning and click scans.

**Comment by oceanor** [comment]
I did it and it was working for an older version, then they added the consumer/producer algorythm to the search queue and it stopped working ^^'

**Comment by simoher** [comment]
What this come down to is, not to scan already scanned locations, more than once per xx sek. 
2 different user could for example choose the same location, but there is still no need to rescan.

**Comment by magimat** [comment]
Runtime toggling of scan would be awesome. I have my server running at home and I'd like to be able to remotely stop/start the live scan, because most of the time, I don't wan it to scan. right now I'm simply shutting the whole server on and off, but it would be a lot more convenient to be able to let it sit and re-enable live scanning on demand.
